.changeset/purple-boxes-wash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/adapter-netlify': minor
+---
+
+chore: upgrade to @netlify/functions v2"
```

Comment:
```suggestion
chore: upgrade to @netlify/functions v2
```

Reply:
Does this even need a changeset since it's a dev dependency of adapter-netlify??

Reply:
Ah, no. We should drop the changeset. Good catch!